dcpy/library/config.py

```diff
@@ -36,21 +30,16 @@ def unparsed_unrendered_template(self) -> str:
             return f.read()
 
     @property
-    def parsed_unrendered_template(self) -> dict:
+    def parsed_unrendered_template(self) -> Dataset:
         """parsing unrendered template into a dictionary"""
-        return yaml.safe_load(self.unparsed_unrendered_template)
+        return Dataset(**yaml.safe_load(self.unparsed_unrendered_template)["dataset"])
 
-    def parsed_rendered_template(self, **kwargs) -> dict:
+    def parsed_rendered_template(self, version: str) -> Dataset:
```

Comment:
We never use other kwargs here. Debated getting rid of this entirely, but it's convenient when path requires version specified